Documentation/ceph-csi-troubleshooting.md

```diff
@@ -0,0 +1,317 @@
+---
+title: Ceph-CSI Configuration
+weight: 1200
+indent: true
+---
+
+# Troubleshooting Ceph-CSI
+
+The issue in Provisioning the CephFS/RBD PVC or Mounting the CephFS/RBD PVC to application pods can happen for Many reasons like Network connectivity between CSI pods and ceph, cluster health issue, Slow operations, kubernetes issue, etc. Even the issue can be at the Ceph-CSI layer itself. Hope the below-documented steps might help you to understand and Debug the ceph-csi issues.
```

Comment:
Do you have the Code Spell Checker plugin for vscode? It will help catch some common spelling and grammar issues.

Reply:
I used grammarly(web plugins) for this same reason but I'll check with vscode extension.